src/modules/flow/oic/Makefile

```diff
@@ -1,4 +1,5 @@
 obj-$(FLOW_NODE_TYPE_OIC) += oic.mod
 obj-oic-$(FLOW_NODE_TYPE_OIC) := oic.json oic.o
-obj-oic-$(FLOW_NODE_TYPE_OIC)-oic := core.light.json
+obj-oic-$(FLOW_NODE_TYPE_OIC)-oic := y
+obj-oic-$(FLOW_NODE_TYPE_OIC)-oic-dirs := $(OIC_DATA_MODELS)
```

Comment:
If there is any way to represent the current dir all of them could be used on oic-dirs.

But I believe just using

obj-oic-$(FLOW_NODE_TYPE_OIC)-oic := y

would be more frequent than including extra dirs anyway.